zephyr/esp32s3/CMakeLists.txt

```diff
@@ -145,7 +147,7 @@ if(CONFIG_SOC_SERIES_ESP32S3)
     ../../components/spi_flash/spi_flash_hpm_enable.c
     ../../components/spi_flash/spi_flash_os_func_noos.c
     ../../components/spi_flash/spi_flash_os_func_app.c
-    # todo: ../../components/spi_flash/spi_flash_wrap.c
+    ../../components/spi_flash/spi_flash_wrap.c
```

Comment:
Looks like this is the only change in this commit which I had tried back in June (see https://github.com/zephyrproject-rtos/zephyr/issues/73677#issuecomment-2160147761).  If this works now, then it seems that the root cause was fixed in some other changes that have already been merged.

Reply:
In fact, this change will only fix the issue together with this other PR being merged  on the Zephyr side:
https://github.com/zephyrproject-rtos/zephyr/pull/79286